packages/components/src/query-controls/README.md

```diff
@@ -66,24 +66,33 @@ The `QueryControls` component now supports multiple category selection, to repla
 const QUERY_DEFAULTS = {
 	orderBy: 'title',
 	order: 'asc',
-	selectedCategories: [ 1 ],
-	categories: [
+	selectedCategories: [
+		{
+			id: 1,
+			value: 'Category 1',
+		},
 		{
+			id: 2,
+			value: 'Category 1b',
+		},
```

Comment:
Let's add the full category object, to make the example more complete

```suggestion
		{
			id: 1,
			value: 'Category 1',
			parent: 0,
		},
		{
			id: 2,
			value: 'Category 1b',
			parent: 1,
		},
```

Reply:
Ah, right!